Sources/Widgets/Core/WidgetManager/Constants.d.ts

```diff
@@ -0,0 +1,25 @@
+export declare const ViewTypes = {
```

Comment:
error TS1254: A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.